configuration_schemas/v4/host_resolution.json

```diff
@@ -0,0 +1,23 @@
+{
+    "id": "file:///var/lib/irods/configuration_schemas/v4/host_resolution.json",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "properties": {
+        "host_entries": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "address_type": {"enum": ["local","remote"]},
+                    "addresses": {
+                        "type": "array",
+                        "items": {"type": "string"},
+                        "uniqueItems": true
+                    }
```

Comment:
@trel Here is the change that makes `addresses` into an array of strings. I also added the `uniqueItems` option.

Reply:
looks good on initial inspection.    true test will be the validator.

Reply:
consider also
`"minItems": 2,`
which would make sure there is an entry and at least one 'alias'

Reply:
and potentially the string itself should be `, "minLength": 1`

Reply:
Will those options have a negative effect on an upgrade with an empty hosts_config.json file?

Reply:
no, pretty sure we're safe.   there is no requirement that a host_entry exists... but if one does...

minItems is for if the array exists
and
minLength is for the strings themselves

Reply:
Got it

Reply:
done.